googlemock/include/gmock/gmock-matchers.h

```diff
@@ -2650,7 +2650,7 @@ class QuantifierMatcherImpl : public MatcherInterface<Container> {
   template <typename InnerMatcher>
   explicit QuantifierMatcherImpl(InnerMatcher inner_matcher)
       : inner_matcher_(
-           testing::SafeMatcherCast<const Element&>(inner_matcher)) {}
+           ::testing::SafeMatcherCast<const Element&>(inner_matcher)) {}
```

Comment:
To quote https://abseil.io/tips/119:
> 1) Never declare namespace aliases or convenience using-declarations at namespace scope in header files, only in `.cc` files.
> 2) Declare namespace aliases and using-declarations inside the innermost namespace, whether named or anonymous. (Do not add an anonymous namespace just for this purpose.)
> 3) When declaring namespace aliases and using-declarations, use fully qualified names (with leading `::`) unless you are referring to a name inside the current namespace.
> 4) For other uses of names, avoid fully qualifying when reasonable

This would fall under (4), so we would prefer not to fully qualify `SafeMatcherCast` here.

Reply:
Thank you for sharing the tips.
I have modified source code as follows referring the tips.
Please let me know if there are any mistakes.

1. source code (inside the `testing` namespace)
`using testing::xxx`
`testing::xxx`

2. source code (outside the `testing` namespace), comments, and documents
`using ::testing::xxx`
`::testing::xxx`